Evaluation Templates

jlopez8 opened the pull request:
This PR includes directories for mathematical equations used to evaluate the Math Notes tool.  Directory `inputs` contains ~98 `.png` files of math images drawn using the canvas tool to be used in this project and sent to the MathPix API. It also contains `inputs.csv`, which stores the corresponding LaTeX. Directory `outputs` contains the rendered LaTeX using the content returned by MathPix. The literal MathPix returns are stored in `outputs.csv`.

rkingery reviewed (changes requested):
Two comments:

1. Fix the name of this file to align with the others (or remove it)
![image](https://user-images.githubusercontent.com/29710965/123754575-0cb15700-d889-11eb-8d3f-4283dc593677.png)

2. Github is loading *very* slowly when trying to do this code review, likely due to all the images you added. I *highly* recommend seeing how much memory this new dataset is taking up before merging. You don't want this dataset to slow down future attempts to use git with this repo, and it will if it's having to download and track a lot of data. Certainly make sure it's not taking up more than 1 GB, preferably under 100 MB. If the dataset does turn out to be "big", you have two options (at least): (a) Find a way to compress the images to be smaller (e.g. using JPEGs instead of PNGs and possibly even zipping up those folders. (b) Keep the data in a separate location online and access it via web commands (wget, gdrive, curl, etc).

jlopez8 commented:
> Two comments:
> 
>     1. Fix the name of this file to align with the others (or remove it)
>        ![image](https://user-images.githubusercontent.com/29710965/123754575-0cb15700-d889-11eb-8d3f-4283dc593677.png)

Fixed. Removed redundant file.

> 
>     2. Github is loading _very_ slowly when trying to do this code review, likely due to all the images you added. I _highly_ recommend seeing how much memory this new dataset is taking up before merging. You don't want this dataset to slow down future attempts to use git with this repo, and it will if it's having to download and track a lot of data. Certainly make sure it's not taking up more than 1 GB, preferably under 100 MB. If the dataset does turn out to be "big", you have two options (at least): (a) Find a way to compress the images to be smaller (e.g. using JPEGs instead of PNGs and possibly even zipping up those folders. (b) Keep the data in a separate location online and access it via web commands (wget, gdrive, curl, etc).

Investigating now.

jlopez8 commented:
Looking at the subdirectories, the source files folder that has the original clips for the formulas selected for evaluation is large ~ 7MBs compared to the rest. I'll see about translating converting these to JPG, and if it's still relatively large, I'll add this directory to .gitignore.
